components/Dialogs/examples/DialogsCustomShadowViewController.swift

```diff
@@ -0,0 +1,111 @@
+
+    bodyLabel.text =
+      "This presented view has a corner radius so we've set the corner radius on the presentation controller."
+    bodyLabel.translatesAutoresizingMaskIntoConstraints = false
+    bodyLabel.numberOfLines = 0
+    bodyLabel.sizeToFit()
+    self.view.addSubview(bodyLabel)
+
+    NSLayoutConstraint.activate(
+      NSLayoutConstraint.constraints(withVisualFormat: "H:|-[body]-|", options: [], metrics: nil, views: [ "body": bodyLabel]))
+    NSLayoutConstraint.activate(
+      NSLayoutConstraint.constraints(withVisualFormat: "V:|-[body]-|", options: [], metrics: nil, views: [ "body": bodyLabel]))
+  }
+
+  override var preferredContentSize: CGSize {
+    get {
+      return CGSize(width:200.0, height:140.0);
+    }
+    set {
+      super.preferredContentSize = newValue
+    }
+  }
+
+}
+
+class DialogsCustomShadowViewController: UIViewController {
+
+  let flatButton = MDCFlatButton()
+  let transitionController = MDCDialogTransitionController()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    view.backgroundColor = UIColor.white
+
+    flatButton.setTitle("PRESENT ALERT", for: UIControlState())
+    flatButton.setTitleColor(UIColor(white: 0.1, alpha:1), for: UIControlState())
+    flatButton.sizeToFit()
+    flatButton.translatesAutoresizingMaskIntoConstraints = false
+    flatButton.addTarget(self, action: #selector(tap), for: .touchUpInside)
+    self.view.addSubview(flatButton)
+
+    NSLayoutConstraint.activate([
+      NSLayoutConstraint(item:flatButton,
+                       attribute:.centerX,
+                       relatedBy:.equal,
+                       toItem:self.view,
+                       attribute:.centerX,
+                       multiplier:1.0,
+                       constant: 0.0),
+      NSLayoutConstraint(item:flatButton,
+                       attribute:.centerY,
+                       relatedBy:.equal,
+                       toItem:self.view,
+                       attribute:.centerY,
+                       multiplier:1.0,
+                       constant: 0.0)
+    ])
+  }
+
+  @objc func tap(_ sender: Any) {
+    let presentedController = CustomShadowViewController(nibName: nil, bundle: nil)
+    presentedController.modalPresentationStyle = .custom;
+    presentedController.transitioningDelegate = self.transitionController;
+
+    self.present(presentedController, animated: true, completion: nil)
+
+    // We set the corner radius to adjust the shadow that is implemented via the trackingView in the
+    // presentation controller.
+    if let presentationController = presentedController.mdc_dialogPresentationController {
+      presentationController.trackingViewCornerRadius = presentedController.view.layer.cornerRadius
+    }
+  }
+
+  // MARK: Catalog by convention
+  @objc class func catalogBreadcrumbs() -> [String] {
+    return [ "Dialogs", "View with Corner Radius"]
+  }
+
+  @objc class func catalogIsPrimaryDemo() -> Bool {
+    return false
+  }
+
```

Comment:
I believe we've been following the convention of explicitly implementing the catalog by convention APIs. In this `catalogIsPresentable` is most notably missing and should likely explicitly return false.

Reply:
Done.